packages/salesforcedx-vscode-apex-debugger/package.nls.ja.json

```
@@ -2,7 +2,7 @@
   "user_id_filter_text": "デバッグ用にホワイトリストに登録される、リクエストを含むユーザー ID の一覧",
   "request_type_filter_text": "デバッグ用にホワイトリストに登録される Apex 要求種別の一覧",
   "entry_point_filter_text": "デバッグ用にホワイトリストに登録される、エントリポイントに一致する正規表現",
-  "sfdx_project_text": "SFDX プロジェクトの場所",
+  "salesforce_project_text": "SFDX プロジェクトの場所",
```

To match the English

```suggestion
  "salesforce_project_text": "Salesforce プロジェクトの場所",
```